tornado/test/options_test.py

```diff
@@ -24,6 +24,13 @@
         mock = None
 
 
+class email(str):
```

Comment:
Class names [should be capitalized](https://www.python.org/dev/peps/pep-0008/#class-names)

Subclassing `str` is subtle; is there some reason why it needs to be a subclass of `str` instead of just a plain class with a `value` attribute? Or you could make `email` a factory function instead of a class: in most cases all you need to do is validate the value and return a string; it doesn't need to stay a distinct class.